app/screens/edit_profile/edit_profile.js

```diff
@@ -92,16 +92,13 @@ export default class EditProfile extends PureComponent {
         currentUser: PropTypes.object.isRequired,
         navigator: PropTypes.object.isRequired,
         theme: PropTypes.object.isRequired,
+        commandType: PropTypes.string.isRequired,
```

Comment:
I'm not seeing where this is assigned other than in the test

Reply:
@enahum RNN v1 internally assigned `commandType` in Android and iOS native codes.
However, I've found that they removed `commandType` in RNN v2, so this will not code with RNN v2.
Well, assigning `commandType` manually can fix this, will be this enough?

Reply:
yup manually assign will be ok I guess as we are actually migrating from RNN v1 to v2

Reply:
@enahum Please check for the changes.